runtime/runtime-params-estimator/src/main.rs

```diff
@@ -26,9 +27,12 @@ struct CliArgs {
     /// How many iterations per block are we going to try.
     #[clap(long, default_value = "10")]
     iters: usize,
-    /// How many accounts were generated with `genesis-populate`.
+    /// Number of active accounts in the state (accounts used for estimation).
     #[clap(long, default_value = "10000")]
     accounts_num: usize,
+    /// Number of additional accounts to add to the state, among which active accounts are selected.
+    #[clap(long)]
+    additional_accounts_num: Option<usize>,
```

Comment:
It feels like this should have a `deafult_value` -- we always want to populate genesis with some amount of additional accounts, no?

Reply:
The problem is that we want to execute it before launching docker, and after that, we need to drop this argument, because we shouldn't populate genesis state twice. `Option` seems to be the simplest solution here.

Reply:
Good question. Let's just use `0` as "don't run genesis populaet", and add that arg to `--docker` mode? If we need this elsewhere, we might add an extra flag for this. 

I do think that we need a  default here, so that run without args does the right thing.